core/src/com/mygdx/game/Components/Pirate.java

```diff
@@ -80,10 +80,10 @@ public void shoot(Vector2 dir) {
     /**
      * Adds ammo
      *
-     * @param ammo amount to add
+     * @param newAmmo amount to add
      */
-    public void reload(int ammo) {
-        this.ammo += ammo;
+    public static void addAmmo(int newAmmo) {
```

Comment:
Probably a good idea changing this but would it be better to simply have a setter?

Reply:
Yeah I thought that too; only renamed their existing function here so it matched with similar functions.

(e.g. 'addPlunder' and 'addHealth')